src/main/java/com/flansmod/common/PlayerHandler.java

```diff
@@ -37,6 +38,8 @@
     private static final Random rand = new Random();
     public static Map<String, PlayerData> serverSideData = new HashMap<String, PlayerData>();
     public static Map<String, PlayerData> clientSideData = new HashMap<String, PlayerData>();
+    public static Map<String, PlayerInfo> serverSideInfo = new HashMap<String, PlayerInfo>();
```

Comment:
I think PlayerInfo needs to be renamed to "PlayerStatistics" or "PlayerTeamsStatistics", and change serverSideInfo to serverSidePlayerStats to make it clear what the difference to serverSideData is

Reply:
okay

Reply:
do I need to change all the files again?

Reply:
This has been resolved.